react/features/mobile/external-api/middleware.js

```diff
@@ -82,7 +82,7 @@ MiddlewareRegistry.register(store => next => action => {
                 store,
                 CONFERENCE_TERMINATED,
                 /* data */ {
-                    url: toURLString(locationURL)
```

Comment:
These seem to be a really big change, or is it just me? Can it break anything?

Reply:
Not really. It’s part of what we use for the recent list. It skips the token and config options.